<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;#618E13&quot;;</string>
  </property>
  <widget class="QPushButton" name="id_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white;</string>
   </property>
   <property name="text">
    <string>Forgot ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="password_button">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>410</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white</string>
   </property>
   <property name="text">
    <string>Forgot Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;ID : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;Password : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_area">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>102</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_area">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>182</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="main_page_button">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>410</y>
     <width>170</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: solid white;</string>
   </property>
   <property name="text">
    <string>Main Page</string>
   </property>
  </widget>
  <zorder>id_button</zorder>
  <zorder>password_button</zorder>
  <zorder>login_button</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>id_area</zorder>
  <zorder>main_page_button</zorder>
  <zorder>password_area</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
